app/views/assets/_asset_checkbox_row.html.erb

```diff
@@ -0,0 +1,72 @@
+  show_permissions=true
+  permissions_selector ||="div\##{item.class.name}_#{item.id}.permission_list"
+
+-%>
+<div class="isa-tree <%= html_classes -%> <%= tree_class %> _<%= item.id %>">
+  <% if toggle %>
+    <%= render :partial => 'assets/isa-tree-toggle', locals: { cb_parent_selector: cb_parent_selector } -%>
+  <% end %>
+  <% if item.can_view? %>
+    <div  style="display: inline; margin-right:1em">
+      <%= render :partial => 'general/split_button_checkbox',
+                 locals: { checkbox_id: publishing_item_param(item),
+                           checkbox_class: "#{item.class.name}_#{item.id}",
+                           checked: checked,
+                           checkbox_text: "",
+                           published: (item.is_published? && block_published),
+                           cant_manage: !(item.can_manage?),
+                           toggle: toggle,
+                           cb_parent_selector: cb_parent_selector} -%>
+    </div>
+    <h5 class="type_and_title" style="display: inline">
+      <strong><%= text_for_resource item -%>:</strong> <%= link_to item.title, item, :target => "_blank" -%>
+      <a class="btn btn-default permissions_toggle" style="display: inline; margin-left:2em; padding:6px"
+          data-permissions_selector='<%= permissions_selector %>'>
+        <%=list_item_visibility(item)-%>
+      </a>
+    </h5>
+  <% else %>
+    <div  style="display: inline; margin-right:1em">
+      <%= render :partial => 'general/split_button_checkbox',
+                 locals: { checkbox_id: publishing_item_param(item),
+                           checkbox_class: "#{item.class.name}_#{item.id}",
+                           not_visible: true,
+                           toggle: toggle,
+                           cb_parent_selector: cb_parent_selector} -%>
+    </div>
+    <h5 class="type_and_title" style="display: inline"><strong><%= text_for_resource item -%>:</strong> This item is hidden to you</h5>
+    <%=list_item_visibility(item)-%>
+    <% show_permissions=false%>
+    <% unless current_user.try(:person) && item.can_see_hidden_item?(current_user.person) %>
+      <% show_managers=false%>
+    <% end %>
+  <% end %>
+  <% if show_managers %>
+    <a class="btn btn-default managed_by_toggle" style="display: inline; margin-left:1em; padding:6px"
+        data-managed_by_selector='<%= managed_by_selector %>'>
+      <span class="glyphicon glyphicon-user" aria-hidden="true" style="position:relative; top:2px;" ></span>
+    </a>
+    <div class="managed_by_list" id=<%="#{item.class.name}_#{item.id}"-%> style="display:none;margin-left:5em">
+      <strong>Manageable by:</strong> <%= item.managers.empty? ? "<span class='none_text'>None</span>".html_safe : item.managers.collect { |m| link_to(h(m.title), m) }.join(", ").html_safe -%>
+    </div>
+  <% end %>
+  <% if show_permissions %>
+      <div class="permission_list" id=<%="#{item.class.name}_#{item.id}"-%> style="display:none;margin-left:5em" >
```

Comment:
`id` should be wrapped in quotation marks `id="<% ...` (even though it will probably work without)

Reply:
Fixed in [b38a156](https://github.com/seek4science/seek/pull/1551/commits/b38a156219a6e4215e7777b77eb7fb4d9dca351c).